Frontend/src/pages/test/login.jsx

```diff
@@ -4,7 +4,11 @@ import { USER_KEY } from '../../ui/providers/UserProvider'
 
 export default function TestLogin() {
   const { login_id } = useParams()
-  useEffect(() => localStorage.setItem(USER_KEY, login_id), [login_id])
+  useEffect(() => {
+    localStorage.setItem(USER_KEY, login_id)
+    // reload the page so the SPA notice that we are 'logged in'
+    window.location = '/'
+  }, [login_id])
```

Comment:
Where is this component used again?
Reloading the page seems like a red flag, even if it's 'just' for a test.

Reply:
It is used to "login" to test users by just setting the localstorage instead of setting the user via /users/me. This is because I don't have the localstorage user key as a use effect in the User Context (because it is just for testing)